test/lit_tests/batchtests/autodiff.mlir

```diff
@@ -0,0 +1,11 @@
+module {
+  func.func @f(%arg0: tensor<f32>) -> tensor<f32> {
+    %0 = stablehlo.multiply %arg0, %arg0 : tensor<f32>
+    return %0 : tensor<f32>
+  }
+
+  func.func @main(%arg0: tensor<10xf32>) -> tensor<10xf32> {
+    %0 = enzyme.batch @f(%arg0) { batch_shape = array<i64: 10> } : (tensor<10xf32>) -> tensor<10xf32>
+    return %0 : tensor<10xf32>
+  }
+}
```

Comment:
can you add the test checks?

Reply:
ah sorry I did not mean to include this in this pr. I will open up another PR that bumps Enzyme + JAX with this test.